commands/actions/publish.go

```diff
@@ -125,6 +126,7 @@ func buildFunc(cmd *cobra.Command, args []string) {
 		outDir = filepath.Join(outDir, *tsconfig.CompilerOptions.OutDir)
 		mustInstallDependencies(actions.Sources)
 		mustBuildProject(actions.Sources, tsconfig)
+		mustExistCompiledFiles(outDir, actions)
```

Comment:
Can we somehow handle this, instead of forcing the user to put everything inside of the `actions` folder?